src/assets/images/Icon/sort-order-arrow.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 96 96" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="Icons_Play" overflow="hidden"><path d="M19.3 11 76.7 48 19.3 85Z"/></svg>
```

Comment:
there is no new line at the end of file .

Reply:
done